.github/workflows/run_test_and_examples.yml

```diff
@@ -8,8 +8,6 @@ jobs:
           - os: ubuntu-22.04
             compiler: gcc-13
 
-          - os: ubuntu-22.04
-            compiler: clang-15
```

Comment:
Supporting it was additional effort because it apparently doesn't generate automatic constructors properly. See https://github.com/rdf4cpp/rdf4cpp/actions/runs/8063550056/job/22025596346